Command/LoadDataFixturesDoctrineODMCommand.php

```diff
@@ -82,37 +84,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        if ($dirOrFile) {
-            $paths = is_array($dirOrFile) ? $dirOrFile : [$dirOrFile];
-        } elseif ($bundles) {
-            $paths = [$this->getKernel()->getRootDir().'/DataFixtures/MongoDB'];
-            foreach ($bundles as $bundle) {
-                $paths[] = $this->getKernel()->getBundle($bundle)->getPath();
-            }
-        } else {
-            $paths = $this->container->getParameter('doctrine_mongodb.odm.fixtures_dirs');
-            $paths = is_array($paths) ? $paths : [$paths];
-            $paths[] = $this->getKernel()->getRootDir().'/DataFixtures/MongoDB';
-            foreach ($this->getKernel()->getBundles() as $bundle) {
-                $paths[] = $bundle->getPath().'/DataFixtures/MongoDB';
-            }
-        }
+        $groups   = $input->getOption('group');
+        $fixtures = $this->fixturesLoader->getFixtures($groups);
+        if (! $fixtures) {
+            $message = 'Could not find any fixture services to load';
 
-        $loaderClass = $this->container->getParameter('doctrine_mongodb.odm.fixture_loader');
```

Comment:
Dropping this is a BC break: we can change the default value in 3.x but not drop it entirely.

Reply:
@alcaeus 
Should i add an --autowire option to the command, which if provided it uses the new symfonyLoader?

And if not provided, it uses the old way with the --bundles or --fixtures option? 
And a check that if you pass --autowire, you can't pass --bundles or --fxitures?
And add an @deprecated if you use the old 2? 

Then in 4.0 you can remove the --bundles,--fixtures and old loader?

Reply:
The more I think about it, the more I think we need to move this into 4.0. Then you can break BC as much as you want...

Reply:
The problem is that i can't easily move to 4.0 as lots of code breaks due to the odm2.0 chnges. So if there is a way to get this onto 3.x i can already clean up my fixtures. 
I'll see if i can find an easy way to have both approaches working

Reply:
Or maybe add a new fixture loader command and deprecate the old one, where in 4.0 you can only use the new one?